proto/palomachain/paloma/valset/snapshot.proto

```diff
@@ -11,7 +11,7 @@ option go_package = "github.com/palomachain/paloma/x/valset/types";
 
 message Validator {
   bytes shareCount = 1 [
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.customtype) = "cosmossdk.io/math.Int",
```

Comment:
Why valset proto files updated in this PR?

Reply:
@aleem1314  because we are using it the evm module.